app/src/components/fields/fieldErrorHint/fieldErrorHint.jsx

```diff
@@ -267,39 +272,34 @@ export class FieldErrorHint extends Component {
       error,
       active,
       staticHint,
+      staticTouchedHint,
       widthContent,
       darkView,
-      provideHint,
       ...rest
     } = this.props;
     const classes = cx('field-error-hint', `type-${hintType}`);
 
     return (
       <div className={classes}>
-        {children &&
-          cloneElement(children, {
-            error: error && messages[error] ? intl.formatMessage(messages[error]) : error,
-            active,
-            ...rest,
+        {children && cloneElement(children, { error, active, ...rest })}
+        <div
+          className={cx('hint', `type-${hintType}`, {
+            'static-hint': staticHint,
+            staticTouchedHint,
+            visible: this.isHintVisible(),
           })}
-        {provideHint && (
+        >
           <div
-            className={cx('hint', `type-${hintType}`, {
+            className={cx('hint-content', `type-${hintType}`, {
               'static-hint': staticHint,
-              visible: this.isHintVisible(),
+              'static-touched-hint': staticTouchedHint,
```

Comment:
Parent of this element already have necessary class, can we relay on it and use nesting?

Reply:
As I checked no